.travis.yml

```diff
@@ -74,9 +76,10 @@ before_install:
         export PATH=/usr/local/opt/flex/bin:/usr/local/opt/bison/bin:$PATH;
         brew unlink python@2;
         brew link --overwrite python;
+        python3 -m venv .venv && source .venv/bin/activate;
     else
         pyenv global $PYTHON_VERSION;
-        ls /usr/bin/;
+        python3 -m venv .venv && source .venv/bin/activate;
```

Comment:
just curious - is it necessary to create virtual env?

Reply:
@ohm314 ?

Reply:
No, we do without too, but I find this cleaner and more flexible. Also having wheels in mind I think it's better to build within a venv rather than having the "system's" environment.